chromium_src/components/vector_icons/aggregate_vector_icons.py

```diff
@@ -4,60 +4,90 @@
 # You can obtain one at https://mozilla.org/MPL/2.0/.
 
 import json
-from pathlib import Path
 import os
+import shlex
+import shutil
 
+import brave_chromium_utils
+import override_utils
 
-def get_root():
-    return f'{Path(__file__).parent}/../..'
+VERBOSE = False  # Set to True to print verbose messages.
 
 
-def get_root_relative(icon_path):
-    return "//" + os.path.relpath(os.path.abspath(icon_path), get_root()) \
-      .replace(os.sep, '/')
+# Loads the leo overrides from leo_overrides.json.
+def LoadLeoOverrides():
```

Comment:
Hey, not sure if we care or not, but I don't think Chromium uses CamelCase any more in their python style:
https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/python/python.md#our-previous-python-style

They say to keep using it in existing files which use it, but going forward to use `unix_hacker_style`